### ISSUE TITLE
`valid-jsdoc` rule: avoid erring for missing `@param`'s or `@return(s)` with `@implements` on the block

### ISSUE THREAD

**Issue by brettz9** [eslint/eslint#11037]
**Tell us about your environment**

* **ESLint Version:** 5.8.0
* **Node Version:** 9.2.1
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
    "parserOptions": {
        "ecmaVersion": 5,
        "sourceType": "script",
        "ecmaFeatures": {}
    },
    "rules": {
        "valid-jsdoc": 2
    },
    "env": {}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

https://eslint.org/demo/#eyJ0ZXh0IjoiLyoqXG4gKiBAaW50ZXJmYWNlXG4gKiBAcGFyYW0ge3N0cmluZ30gbmFtZSBUaGUgY29sb3IgbmFtZVxuICogQHJldHVybnMge29iamVjdH0gQSBjb2xvciBvYmplY3RcbiAqL1xuZnVuY3Rpb24gQ29sb3IgKG5hbWUpIHt9XG5cblxuLyoqXG4gKiBGdW5jdGlvbiByZXByZXNlbnRpbmcgYSB0cmFuc3BhcmVudCBjb2xvci5cbiAqIEBpbXBsZW1lbnRzIHtDb2xvcn1cbiAqL1xuZnVuY3Rpb24gVHJhbnNwYXJlbnRDb2xvcihhKSB7fVxuIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo1LCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7InZhbGlkLWpzZG9jIjoyfSwiZW52Ijp7fX19

```js
/**
 * @interface
 * @param {string} name The color name
 * @returns {object} A color object
 */
function Color (name) {}


/**
 * Function representing a transparent color.
 * @implements {Color}
 */
function TransparentColor(a) {}

```

No special command used to run ESLint here; just the demo (though getting it from `eslint .` locally as well).

**What did you expect to happen?**

There should be no error.

The [JSDocs (for @implements)](http://usejsdoc.org/tags-implements.html) states:

> If you do not document one of the symbols in the implementation, JSDoc will automatically use the interface's documentation for that symbol.

This would suggest that it is expected that with `@implements` present, values need only be added if they differ from the interface they are implementing.

While JSDoc even states:

> You do not need to add the @implements tag to each member of the implementation (for example, the implementation's instance methods).

...it would be helpful if ESLint could least avoid the error when `@implements` is explicit on the block. In SVGEdit, we have a number of these real-world uses which are getting unduly flagged, and it would be a maintenance head-ache to have to duplicate parameter and return information on implementing objects as well as on the interface.

([@augments](http://usejsdoc.org/tags-augments.html) should probably behave this way as well.)

**What actually happened? Please include the actual, raw output from ESLint.**

> 2:5 - JSDoc syntax error. (valid-jsdoc)

**Are you willing to submit a pull request to fix this bug?**

No, not for the foreseeable future.

**Comment by nzakas** [comment]
Thanks for the issue. Similar to #11036, this is a parsing error caused by Doctrine, not actually part of ESLint itself. Can you [open an issue](https://github.com/eslint/doctrine/issues/new) there instead?
